topdown/graphql.go

```diff
@@ -85,6 +85,36 @@ func validateQuery(schema *gqlast.Schema, query *gqlast.QueryDocument) error {
 	return nil
 }
 
+func getBuiltinSchema() (*gqlast.SchemaDocument, error) {
+	schema, err := gqlparser.ParseSchema(gqlvalidator.Prelude)
+	if err != nil {
+		return nil, fmt.Errorf("Error in gqlparser Prelude (should be impossible): %s", err.Message)
```

Comment:
I was wondering if this was possible 😄 

So we could also make it
```go
func getBuiltinSchema() *gqlast.SchemaDocument
```
and `panic` if err is not nil.